.github/workflows/build_and_publish.yml

```diff
@@ -10,15 +10,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+
+      - name: Load AWS credentials
+        id: load-aws-credentials
+        uses: 1password/load-secrets-action@v1
         with:
-          python-version: '2.7'
-      # Setup .npmrc file to publish to npm
+          export-env: true
+        env:
+          OP_CONNECT_HOST: ${{ secrets.OP_CONNECT_HOST }}
+          OP_CONNECT_TOKEN: ${{ secrets.OP_CONNECT_TOKEN }}
+          NPM_TOKEN: op://CI-CD/NPM-datafoldnpm-Token/credential
+
       - uses: actions/setup-node@v3
         with:
           node-version: '10.x'
```

Comment:
Probably, GHA will complain about this (they don't like node <12), but let's try and see